ideascube/serveradmin/tests/test_views.py

```diff
@@ -231,20 +231,20 @@ def test_backup_should_list_available_backups(staffapp, backup):
     form = staffapp.get(reverse('server:backup')).forms['backup']
     radio_options = form.get('backup').options
     assert len(radio_options) == 4
-    assert radio_options[3][0] == backup.name
+    assert radio_options[2][0] == backup.name
 
 
 def test_backup_button_should_save_a_new_backup(staffapp, monkeypatch,
                                                 settings):
-    monkeypatch.setattr('ideascube.serveradmin.backup.Backup.FORMAT', 'zip')
+    monkeypatch.setattr('ideascube.serveradmin.backup.Backup.FORMAT', 'gztar')
```

Comment:
Having to monkeypatch this makes me think it probably shouldn't be a class attribute in the first place...